contrib/custom_top_level_templates.go

```diff
@@ -28,8 +28,9 @@ func (p CrossGroupTemplateParameters) constructTemplate(params CrossGroupTemplat
 		panic(err)
 	}
 	inputSnapshotTemplate := model.CustomTemplates{
-		Templates: map[string]string{params.OutputFilename: templateContents},
-		Funcs:     funcs.MakeTopLevelFuncs(params.OutputFilename, params.SelectFromGroups, params.ResourcesToSelect),
+		Templates:        map[string]string{params.OutputFilename: templateContents},
+		MockgenDirective: true,
```

Comment:
I wasn't sure whether we want to expose `MockgenDirective` as an option for the user of these custom top level templates, so I enabled by default. Open to other ideas